jupyterlab_ucaip/jupyterlab_ucaip/service.py

```diff
@@ -404,11 +404,23 @@ def get_endpoints(self, model_id):
         endpoints.append(built)
     return endpoints
 
-  def check_deploying(self, model_name):
+  def get_all_endpoints(self):
+    gcp_endpoints = self._endpoint_client.list_endpoints(parent=self._parent).endpoints
+    endpoints = []
+    for endpoint in gcp_endpoints:
+      built = self._build_endpoint("None", endpoint)
+      endpoints.append(built)
+    return endpoints
+
+  def check_deploying(self, model_name, endpoint_id):
     endpoints = self._endpoint_client.list_endpoints(parent=self._parent).endpoints
-    name = "ucaip-extension/" + model_name
-    filtered = filter(lambda x: name == x.display_name, endpoints)
-    filtered = list(filtered)
+    if endpoint_id != "":
```

Comment:
should this be just:

```Python
if endpoint_id:
```

Reply:
yup! cool I didn't know you could do that in python